app/assets/javascripts/scroll-func.js

```diff
@@ -0,0 +1,11 @@
+let scrollBtn = document.getElementsByClassName('back-to-top');
+
+window.addEventListener('scroll', function() {
+  let position = window.scrollY;
+
+  if (position < 200) {
+    scrollBtn.header.style.opacity = '0';
```

Comment:
This should be `scrollBtn.style.display="none";`
@NitinBhasneria that's why this doesn't work.